src/components/gataca-qr/gataca-qr.tsx

```diff
@@ -385,19 +384,25 @@ export class GatacaQR {
   }
 
   getLink(): string {
-    if (this.v2) {
+    if (this.v !== "3") {
```

Comment:
comment[suggestion]: I would say this.v === "2" just in case.

Also, as there is a return inside, the else clause is unnecessary